spec/spec.md

```diff
@@ -180,7 +185,20 @@ nested. See [Submission Requirement Rules](#submission-requirement-rules).
 [[ref:Holder]] (via a [[ref:Presentation Definition]]) in order to proceed with
 an interaction.
 
-## Presentation Definition
+## Structure of this Document
+
+Objects are defined such that they may be used on their own or extended through 
+[[ref:Features]] defined subsequently in the spec. A [[ref:Feature]] must 
+declare if it has dependencies on other [[ref:Features]].
+
+A [[ref:Feature]] enables [[ref:Verifiers]] to express, and processing entities
+to support, extended functionality (relative to the base objects) by defining
+one or more properties on one or more objects. 
+
+Processing entities are not required to support [[ref:Features]] on top of the 
+base [[ref:Feature]].
```

Comment:
Thanks @nklomp, these are good questions, and we may need to bring these in as follow-up issues (we're trying to control the size/scope of this PR). Let's discuss tomorrow and figure out if it can be easily addressed in this PR or if it needs to be tracked in a separate issue

Reply:
@nklomp please create issue